include/entity.h

```diff
@@ -353,6 +353,24 @@ typedef struct {
     u8 anim;
 } ET_WeaponUnk014;
 
+typedef struct {
+    s16 unk7C;
+    s16 unk7E;
+    s32 unk80;
+    s16 unk84;
+    s32 : 32;
+    s16 : 16;
+    struct Entity* parent;
```

Comment:
Adding 
```
#ifdef VERSION_PC
u32 padding;
#endif
```
after this point and adding back the assert should fix the issue you were having

Reply:
Hmmm... I can't get it to work. Don't want to blow all the pipeline time so going to play around on my build machine